### PR TITLE
ECF-1-307 admin edits induction coordinators

### PR DESCRIPTION
**Pull request by amazimbe**
### Context
New endpoint to allow admins to edit induction coordinators.

### CODE REVIEW

File: spec/requests/admin/induction_coordinators_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Admin::InductionCoodinators", type: :request do
+  let(:induction_coordinator_profile) { create(:induction_coordinator_profile) }
+  let(:induction_coordinator) { induction_coordinator_profile.user }
+
+  before do
+    admin_user = create(:user, :admin)
```
Comment (seb-siem-softwire) [inline spec/requests/admin/induction_coordinators_spec.rb:10]: You can put it into a let

File: spec/requests/admin/lead_providers/lead_provider_users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Admin::LeadProviders::LeadProviderUsers", type: :request do
+  let(:lead_provider_profile) { create(:lead_provider_profile) }
+  let(:lead_provider_user) { lead_provider_profile.user }
+
+  before do
+    admin_user = create(:user, :admin)
```
Comment (seb-siem-softwire) [inline spec/requests/admin/lead_providers/lead_provider_users_spec.rb:10]: You can put it into a let

**Comment by ethanmills** [comment]
[Allow DFE admins (super user) to edit a user.](https://trello.com/c/b0GGlRLe/307-allow-dfe-admins-super-user-to-edit-a-user)

**Comment by amazimbe** [comment]
Made a mess of a rebase so I have opened a new pr.

**Pull request closed without merging**